data/urdf/wam/wam.urdf

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from /home/js/dev/catkin_workspaces/dev_libherb/src/herb_description/robots/wam_standalone.urdf.xacro | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<robot name="wam" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <link name="world"/>
+  <joint name="/j0" type="fixed">
+    <parent link="world"/>
+    <child link="/wam_base"/>
+    <origin xyz="0 0 0" rpy="0 0 0" />
+  </joint>
+  <link name="/wam_base">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://herb_description/meshes/wam/wam_base.dae"/>
+      </geometry>
+      <material name="">
+        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1"/>
+      </material>
+    </visual>
+    <inertial>
+      <origin rpy="0 0 0" xyz="0.21556578 0.26189039 0.34533511"/>
+      <mass value="9.97059584"/>
+      <inertia ixx="0.10916849" ixy="0.00640270" ixz="0.02557874" iyx="0.00000000" iyy="0.18294303" iyz="0.00161433" izx="0.00000000" izy="0.00000000" izz="0.11760385"/>
+    </inertial>
+    <collision>
+      <geometry>
+        <mesh filename="package://herb_description/meshes/wam/wam_base_collision.STL"/>
+      </geometry>
+    </collision>
+  </link>
+  <link name="/wam1">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://herb_description/meshes/wam/wam1.dae"/>
+      </geometry>
+      <material name="">
+        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1"/>
+      </material>
+    </visual>
+    <inertial>
+      <origin rpy="-1.570796 0 0" xyz="-0.004434 -0.000665 -0.121890"/>
+      <mass value="10.76768767"/>
+      <inertia ixx="0.13488033" ixy="-0.00213041" ixz="-0.00012485" iyx="0.00000000" iyy="0.11328369" iyz="0.00068555" izx="0.00000000" izy="0.00000000" izz="0.09046330"/>
+    </inertial>
+    <collision>
+      <geometry>
+        <mesh filename="package://herb_description/meshes/wam/wam1_collision.STL"/>
+      </geometry>
+    </collision>
+  </link>
+  <joint name="/j1" type="revolute">
+    <origin rpy="-1.1127E-16 0 0" xyz="0.22 0.14 0.346"/>
+    <parent link="/wam_base"/>
+    <child link="/wam1"/>
+    <axis xyz="0 0 1"/>
+    <!-- HERB's J1 joints are modified to have a +180 degree offset. -->
+    <limit effort="1.8" lower="0.54159265359" upper="5.74159265359" velocity="0.75"/>
```

Comment:
If you are going to distribute this file with DART, then I suggest reverting to the default joint limits of the Barrett WAM arm. HERB's WAMs were specially modified by Barrett to have different joint limits, to improve the workspace when two WAMs are mounted side-by-side.